test/e2e_node/mirror_pod_test.go

```diff
@@ -196,6 +198,51 @@ var _ = SIGDescribe("MirrorPod", func() {
 			}, 2*time.Minute, time.Second*4).Should(gomega.BeNil())
 		})
 	})
+
+	ginkgo.Context("static pod admit failed when kubelet start", func() {
+		ginkgo.It("should become Failed and have reason UnexpectedAdmissionError [NodeConformance]", func() {
```

Comment:
```suggestion
		ginkgo.It("should become Failed and have reason UnexpectedAdmissionError [NodeConformance] [Serial]", func() {
```

needs to be serial as you need to update kubelet config as part of the test

Reply:
will try, thanks

Reply:
test ok

Reply:
I think we're still missing `[Serial]` here. Even if the test is okay, I am worried stopping and starting the kubelet mid-test could cause hard to debug flakes